internal/gps/result.go

```diff
@@ -65,6 +68,7 @@ func WriteDepTree(basedir string, l Lock, sm SourceManager, sv bool) error {
 	for _, p := range l.Projects() {
 		to := filepath.FromSlash(filepath.Join(basedir, string(p.Ident().ProjectRoot)))
 
+		logger.Printf("Ensure %v exists", p)
```

Comment:
i think that using "ensure" here may blur the meaning of the word a bit. let's say `"Writing out %s"` here, instead.